### PR TITLE
[WIP] add concating files

### PR DESCRIPTION
**Pull request by mschultheiss83**
renamed requrie.js to autoload
created local folder for configs
added / updated + moved samples

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -120,7 +112,7 @@ module.exports = function(grunt) {
             '!config_room_wallsetter.js',
             '!config_string.js'
           ],
-          dest: 'dist/',
```
Comment (WhiteHalmos) [inline Gruntfile.js]: Prefer to have trailing commas, @TooAngel, WDYT?

Comment (TooAngel) [inline Gruntfile.js]: I have no preference. I understand the reasoning for both.
I like to have it consistent and I like it to be checked automatically. Maybe it is easy to check via jshint / jscs, add it to the config and we are fine.

**Comment by mschultheiss83** [comment]
@TooAngel please take a secoung look

File: src/.gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
-config_local.js
-friends.js
+### local dir
+local/*
+!local/.gitignore
+!local/autoload_config.js
```
Comment (TooAngel) [inline src/.gitignore]: Do we need this, can't find it in the PR

File: src/local/local_config.js.sample [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+//config.profiler.enabled = false;
+//config.stats.enabled = true;
+//config.stats.summary = true;
+//config.layout.plainCost = 4;
+//config.layout.swampCost = 7;
+//config.info.signController = false;
+
+// TODO: need to add config.layout.roadCost
+//config.layout.roadCost = 3;
```
Comment (TooAngel) [inline src/local/local_config.js.sample]: nitpicking: newline

Maybe the files in `src/local` are not jshinted, beautified, jscsed?

File: package.json [hunk 4]
```diff
@@ -1,25 +1,30 @@
 {
   "name": "screeps-bot-tooangel",
-  "version": "1.1.15",
+  "version": "1.2.0",
```
Comment (TooAngel) [inline package.json]: We discussed it a bit in the slack channel. The idea is to announce a release, stop merging PRs. Create a release branch which is tested on private servers and increase the version, when it is fine, release branch is merged, the bot is released. Continue as usual.

I would take this branch as last merge before releasing. So I would suggest to remove the version update and I will start a release after merging and increase the version.

File: Gruntfile.js [hunk 5]
```diff
@@ -89,56 +231,101 @@ module.exports = function(grunt) {
...
+            dest: 'dist/'
+          }, {
+            expand: true,
+            cwd: 'node_modules/screeps-profiler/',
+            src: [
+              'screeps-profiler.js'
+            ],
+            dest: 'build/'
+          }
+          // TODO add docs for screeps-elk!
```
Comment (mschultheiss83) [inline Gruntfile.js]: i ve got n elastic+kibana, would be nice to implement an screeps log shipper to elastic...

Comment (TooAngel) [inline Gruntfile.js]: You mean something like this? https://github.com/Puciek/screeps-elk

We are currently implementing sending stats to grafana screepspl.us. Logging to somewhere would also be nice. Currently I'm too busy with reading PRs :-) to setup something by myself.

So if you like, have a look, add it, I would like it.

**Comment by TooAngel** [comment]
Can you please rebase on master so that I can test.

Also please add the changes, especially regarding the `friends.js` in the README

**Comment by mschultheiss83** [comment]
@TooAngel I'll try it tomorrow

**Comment by Nevrdid** [comment]
You can also join us on slack : https://screeps.slack.com/archives/angels_public
It is easier to share idea and reviews while developing.

**Comment by mschultheiss83** [comment]
Don't have an account on this team yet?
Contact the team administrator for an invitation?
**i didn't get it** _sorry_

**Comment by TooAngel** [comment]
> Don't have an account on this team yet?

http://chat.screeps.com/ You have to register here

**Comment by TooAngel** [comment]
Do you plan to continue work on this?

**Comment by mschultheiss83** [comment]
no i dont, should i remove?

**Pull request closed without merging**